react-components/src/stories/Image360HistroicalOverviewToolbar.stories.ts

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Image360HistoricalOverviewToolbar } from '../components';
+
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
```

Comment:
Are these comments necessary? I think it's okay to omit links to docs, and assume the reader can find it themselves